site/hxapi/hotel/av/index.md

```diff
@@ -2,316 +2,384 @@
- | key            | String  |                  | Y        |                                                                                                                                             |
- | token          | String  | [0-9]{9}         | Y        |                                                                                                                                             |
- | ABTANumber     | String  | [A-Z0-9]{5}      | Y        |                                                                                                                                             |
- | Password       | String  | [A-Z0-9]{5}      | N        | Password required for Agents                                                                                                                |
- | Initials       | String  | [A-Z]{3}         | Y        |                                                                                                                                             |
- | ArrivalDate    | Date    | YYYY-MM-DD       | Y        |                                                                                                                                             |
- | DepartDate     | Date    | YYYY-MM-DD       | N        |                                                                                                                                             |
- | FlightDate     | Date    | YYYY-MM-DD       | N        |                                                                                                                                             |
- | Nights         | Integer | [1,2,3]          | Y        |                                                                                                                                             |
- | RoomType       | String  | [A-Z]{1}[0-9]{2} | Y        | See [RoomCodes](/hxapi/types/roomcode) for a list of valid codes                                                                             |
- | SecondRoomType | String  | [A-Z]{1}[0-9]{2} | N        | See [RoomCodes](/hxapi/types/roomcode) for a list of valid codes                                                                             |
- | ParkingDays    | Integer | [0,8,15]         | N        |                                                                                                                                             |
- | System         | Char    | [A-Z]{3}         | N        | Send ABG if looking for products based in Europe and prices in Euros                                                                        |
- | lang           | Char    | [A-Z]{2}         | N        | Send either en,de,nl,fr,es,pt to return any particular product information is respective language; used only when passing in a fields array |
-                              |
-
-### Request (one room)
+### Endpoint
+
+The endpoint to use is (where "foo" is the location (airport) code):
```

Comment:
Thanks - in parking it still references 'foo' but agree, locationCode is much better. Can go back and change parking separately, will make that change here now.